### DISCUSSION TITLE
`iconMapping` prop for Alerts

### DISCUSSION THREAD

**Discussion by kirillku** [chakra-ui/chakra-ui#8707] [Ideas]
### Description

Would be nice to add `iconMapping` prop for Alerts, so it will be able to set custom icons in theme overrides.

### Problem Statement/Justification

We use custom icons, Chakra UI also have guides how to use custom icons, but alert icons are currently hardcoded. I still wanna use `AlertIcon` but with custom icons.

### Proposed Solution or API

Add `iconsMapping` to `AlertIcon`, so it will be possible to pass it directly or override in theme
```tsx
const AlertIcon = ({ iconsMapping = defaultIconsMapping }) => ...
```

### Alternatives

Alternative is just to make `AlertIcon` component locally, but then we will have two imports with the same name and also someone else might need this feature. 

### Additional Information

I can make a pr

**Comment by segunadebayo** [comment]
`Alert.Indicator` is a slot. Put your icon in it.

You do not need `iconMapping`.
